src/vs/workbench/parts/stats/node/workspaceStats.ts

```diff
@@ -343,36 +399,60 @@ export class WorkspaceStats implements IWorkbenchContribution {
 				tags['workspace.android.cpp'] = true;
 			}
 
-			const packageJsonPromises = !nameSet.has('package.json') ? [] : folders.map(workspaceUri => {
-				const uri = workspaceUri.with({ path: `${workspaceUri.path !== '/' ? workspaceUri.path : ''}/package.json` });
-				return this.fileService.resolveFile(uri).then(() => {
-					return this.fileService.resolveContent(uri, { acceptTextOnly: true }).then(content => {
-						try {
-							const packageJsonContents = JSON.parse(content.value);
-							if (packageJsonContents['dependencies']) {
-								for (let module of ModulesToLookFor) {
-									if ('react-native' === module) {
-										if (packageJsonContents['dependencies'][module]) {
-											tags['workspace.reactNative'] = true;
-										}
-									} else {
-										if (packageJsonContents['dependencies'][module]) {
-											tags['workspace.npm.' + module] = true;
-										}
-									}
+			function getFilePromises(filename, fileService, contentHandler): TPromise<void>[] {
+				return !nameSet.has(filename) ? [] : folders.map(workspaceUri => {
+					const uri = workspaceUri.with({ path: `${workspaceUri.path !== '/' ? workspaceUri.path : ''}/${filename}` });
+					return fileService.resolveFile(uri).then(() => {
+						return fileService.resolveContent(uri, { acceptTextOnly: true }).then(contentHandler);
+					}, err => {
+						// Ignore missing file
+					});
+				});
+			}
+
+			let dependencyFilePromises;
+
+			const requirementsTxtPromises = getFilePromises('requirements.txt', this.fileService, content => {
+				console.log(content.value);
+				let dependencies: string[] = content.value.split('\n');
+				tags['workspace.py.any-azure'] = WorkspaceStats.searchArray(dependencies, /azure/i);
+				for (let dependency of dependencies) {
+					let packageName = dependency.split('==')[0];
+					if (PyModulesToLookFor.indexOf(packageName) > -1) {
```

Comment:
`packageName` should be trimmed before we go looking for it in `PyModulesToLookFor`